src/hooks/useInterval.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useRef } from 'react'
+
+export const useInterval = (cb: () => void, delay: number) => {
+  const cbRef = useRef(() => {
+  })
+
+  useEffect(() => {
+    cbRef.current = cb
+  }, [cb])
+
+  useEffect(() => {
+    const id = setInterval(cbRef.current, delay)
```

Comment:
cbRef doesn't do anything without:
```suggestion
    const id = setInterval(() => cbRef.current(), delay)
```


https://github.com/streamich/react-use/blob/e27c1930da8b94a2c570954786aa68a3dc5624be/src/useInterval.ts#L12

Reply:
How is that any different?

Reply:
added a debug commit to show that it doesn't call the current cb instance
![image](https://github.com/oasisprotocol/dapp-wrose/assets/3758846/2d910b34-10ae-4ab3-8e8c-9cbc9debcab6)

Reply:
Ah, got it. Context is different 👍